lib/engine/game/g_18_fl/step/token.rb

```diff
@@ -83,6 +83,7 @@ def token_cost_override(entity, city_hex, _slot, token)
             node = city_hex.respond_to?(:city?) ? city_hex : city_hex.tile.towns.first
```

Comment:
return unless (node = )

Reply:
Wouldn't that be a syntax error? I could also do `return if !node`

Reply:
it is?

Reply:
I've changed it to `return if node.nil?`

Reply:
return unless node